resources/content/schema/user/user-auth.json

```diff
@@ -130,19 +130,18 @@
         "externalEvent.reportedAccountId": "r",
 
         "externalStoragePoolEvent" : "r",
-        "externalStoragePoolEvent.externalId" : "r",
```

Comment:
You removed these bc they are not needed bc they are specified in the base type?